exporter_test.go

```diff
@@ -639,6 +639,72 @@ version = "4.5.6"
 				})
 			})
 
+			when("checking the image manifest", func() {
+				var fakeRemoteManifestSize int64
+
+				when("platform API is < 0.6", func() {
+					when("image has a manifest", func() {
+						it.Before(func() {
+							fakeRemoteManifestSize = 12345
+							fakeAppImage.SetManifestSize(fakeRemoteManifestSize)
+						})
+
+						it("doesn't set the manifest size in the report.toml", func() {
+							report, err := exporter.Export(opts)
+							h.AssertNil(t, err)
+
+							h.AssertEq(t, report.Image.ManifestSize, int64(0))
+						})
+					})
+				})
+
+				when("platform API is >= 0.6", func() {
+					it.Before(func() {
+						exporter.PlatformAPI = api.MustParse("0.6")
+						exporter.Buildpacks = []lifecycle.GroupBuildpack{
+							// when PlatformAPI is >= 0.5, we need to set Builpacks API
```

Comment:
I think I know why this is true (https://github.com/buildpacks/lifecycle/blob/d9c7de5395c1ba66682b3e548f8955a11b265c17/exporter.go#L442) but maybe it would be better to just set the API when `exporter.Buildpacks` is instantiated, so that we don't have to remember to do it for every future test?

Reply:
Yeah, the line that you posted is the reason. Didn't want to indicate the specific function.
But I agree that it's better to move it to `it.Before` of these unit tests.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/e32d468109f145d40f80ec070f7300d3f807ed80